modules/mw.cx.TargetArticle.js

```
@@ -230,18 +230,20 @@ mw.cx.TargetArticle.prototype.publishSuccess = function (response, jqXHR) {
 		if (this.sourceLanguage === "mdwiki" && publishResult.published_to != "local") {
 			targeturl = publishResult.targeturl_wiki;
 		}
-		var qid;
+		var qid = "";
```

ESLint

[error] Unexpected var, use let or const instead.

(no-var)